Add section about Machine learning

salgo60 opened the pull request.

salgo60 committed:
Add section about Machine learning

consideRatio commented on README.md:
```diff
@@ -16,4 +16,11 @@ With this API you can get events based on _when_, _where_ and _what_, for more d
+* Example SPARQL queries
+    * All [Swedish politicians in the Swedish Goverment](http://tinyurl.com/y4jo4lva)
+        * just those with a [twitter account](http://tinyurl.com/y4qb32q9)
+* In may started also the [Lexicographical data project](https://www.wikidata.org/wiki/Wikidata:Lexicographical_data/en) with the goal to "describe precisely all words in all languages" in a machinereadable form
```
This file is the top level overview of API examples, and will hopefully contain lots of APIs in a year or two. With this in mind, would you agree that it could make sense to:
- put the query examples inside the notebook you made for Wikidata instead, it seems to me more suitable that that contains more extensive details rather than it being in this overview of examples.
- update the description "Wikidata provides machine readable data..." to be as you desire without adding another header?

consideRatio replied:
Ping @salgo60

salgo60 committed:
Jupyter Forsakringskassan 

Dataset Pågående sjukfall per diagnos, specifikt för stressrelaterade sjukdomar 2005–

salgo60 committed:
Added section Försäkringskassan - Get statistics

salgo60 commented:
I add another one with Försäkringskassan
